pyfolio/timeseries.py

```diff
@@ -547,24 +547,21 @@ def rolling_beta(returns, factor_returns,
         return out
 
 
-def rolling_regression(returns, factor_returns=None,
+def rolling_regression(returns, factor_returns,
```

Comment:
Are there still call-sites of this somewhere in PyFolio? I didn't see any after a quick scan.

Reply:
No, but it's possible that users (e.g. Quantopian's research team) depend on this function. In my mind, it is difficult to justify moving this function into, say, `empyrical`, as it is not quite a financial metric, and this sort of calculation usually only arises when analyzing portfolio performance.